js/ui/grid_core/ui.grid_core.search.js

```diff
@@ -279,9 +280,11 @@ module.exports = {
                         $parent = cellElement.parent();
                         if(!$parent.length) {
                             $parent = $("<div>").append(cellElement);
+                        } else if(column) {
+                            findCondition = `td:nth-child(${column.index + 1})`;
```

Comment:
column.index is not a visible index of column.

Reply:
Oh, sure. I've replaced index with visibleIndex.